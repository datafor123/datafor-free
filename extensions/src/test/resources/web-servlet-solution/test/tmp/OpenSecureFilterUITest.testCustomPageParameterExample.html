java.lang.NoClassDefFoundError: org/apache/commons/codec/DecoderException
	at org.apache.commons.httpclient.HttpMethodBase.<init>(HttpMethodBase.java:217)
	at org.apache.commons.httpclient.methods.GetMethod.<init>(GetMethod.java:88)
	at org.pentaho.platform.web.test.ui.BaseUITest.submitGet(BaseUITest.java:90)
	at org.pentaho.platform.web.test.ui.BaseUITest.runUrl(BaseUITest.java:70)
	at org.pentaho.platform.web.test.OpenSecureFilterUITest.runXAction(OpenSecureFilterUITest.java:20)
	at org.pentaho.platform.web.test.OpenSecureFilterUITest.testCustomParameterPageExample(OpenSecureFilterUITest.java:49)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:39)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
	at java.lang.reflect.Method.invoke(Method.java:597)
	at junit.framework.TestCase.runTest(TestCase.java:168)
	at junit.framework.TestCase.runBare(TestCase.java:134)
	at junit.framework.TestResult$1.protect(TestResult.java:110)
	at junit.framework.TestResult.runProtected(TestResult.java:128)
	at junit.framework.TestResult.run(TestResult.java:113)
	at junit.framework.TestCase.run(TestCase.java:124)
	at junit.framework.TestSuite.runTest(TestSuite.java:232)
	at junit.framework.TestSuite.run(TestSuite.java:227)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:81)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:38)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:460)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:673)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:386)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:196)
