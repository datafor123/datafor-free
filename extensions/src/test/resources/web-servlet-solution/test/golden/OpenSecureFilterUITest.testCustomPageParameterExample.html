<html>
   <head>
      <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
      <link rel="stylesheet" type="text/css" href="/pentaho-style/pentaho.css">
      <title>Parameter Page Customization Example</title><script type="text/javascript">

          function dynExec(funcName, formName, objectName) {
            var func = funcName + formName + "('" + objectName + "');";
            var rtn = false;
            try {
              rtn = eval(func);
            } catch (ignored) {
              rtn = false;
            }
            return rtn;
          }

          function setXFormsValue(formName, objectName) {
            // Will cause invoke of setXFormsValueformname(object);
            return dynExec("setXFormsValue", formName, objectName);
          }
          function activate(formName, objectName) {
            // Will cause invoke of activateformname(object);
            return dynExec("activate", formName, objectName);
          }
          function upload(formName, objectName) {
            // Will cause invoke of uploadformname(object);
            return dynExec("upload", formName, objectName);
          }
          </script>
      			
      			<script>
      			  var url=unescape('http://localhost:8080/pentaho/ViewAction?');
      			  var target=unescape('Report_Window');
      			  function doForm() {
      			    var submitUrl = url;
      
      			    var form = document.forms['form'];
      			    var elements = form.elements;
      			    var i;
      			    for( i=0; i<elements.length; i++ ) {
      			      if( elements[i].type == 'select-one' || elements[i].type == 'text' || elements[i].type == 'hidden') {
      			        submitUrl += '&' + elements[ i ].name + '=' + escape( elements[ i ].value );
      			      }
      				      amp = '&';
      			      if( elements[i].type == 'radio' ) {
      			      	if( elements[i].checked ) {
      			          submitUrl += '&' + elements[ i ].name + '=' + escape( elements[ i ].value );
      			      	}
      			      }
      			    }
      			    if( target == '' ) {
      				    document.location.href=submitUrl;
      				} else {
      					window.open( submitUrl, target, '' );
      				}
      			    return false;
      			  }
      			</script>
      		</head>
   <body dir="LTR">
      <div style="margin:10px"><span class="portlet-section-header">Parameter Page Customization Example</span></div>
      <div style="margin:10px;border:1px solid #808080">
         <form name="form">
            <table>
               <tr>
                  <td><span class="portlet-font">This parameter page is generated using a custom (XSL) template. You can create a custom template for each report if you want
                        to...</span></td>
               </tr>
               <tr>
                  <td class="portlet-section-subheader"><br>Select: Region
                  </td>
               </tr>
               <tr>
                  <td class="portlet-font"><input id="REGION" name="REGION" type="text" value="" title="" class="portlet-form-input-field" onchange="javascript:setXFormsValue('custom_parameter_page_example', 'REGION');"><span id="REGION-required" class="required-symbol"></span></td>
               </tr>
               <tr>
                  <td class="portlet-section-subheader"><br>Select: Department
                  </td>
               </tr>
               <tr>
                  <td class="portlet-font"><input id="DEPARTMENT" name="DEPARTMENT" type="text" value="" title="" class="portlet-form-input-field" onchange="javascript:setXFormsValue('custom_parameter_page_example', 'DEPARTMENT');"><span id="DEPARTMENT-required" class="required-symbol"></span></td>
               </tr>
               <tr>
                  <td class="portlet-section-subheader"><br>Select: Report Format
                  </td>
               </tr>
               <tr>
                  <td class="portlet-font"><input id="type" name="type" type="text" value="html" title="" class="portlet-form-input-field" onchange="javascript:setXFormsValue('custom_parameter_page_example', 'type');"><span id="type-required" class="required-symbol"></span></td>
               </tr>
               <tr>
                  <td><br><input type="button" name="go" class="portlet-form-button" value="Update..." onClick="doForm()"></td>
               </tr>
            </table><input xmlns:xf="http://www.w3.org/2002/xforms" type="hidden" name="solution" value="samples"><input xmlns:xf="http://www.w3.org/2002/xforms" type="hidden" name="action" value="custom-parameter-page-example.xaction"><input
             type="hidden" name="path" value="reporting"></form>
      </div>
   </body>
</html>